### PR TITLE
Allow to only get local roles for groups and do not reassign global r…

### PR DESCRIPTION
**Pull request by pbauer**
…oles as local roles

### CODE REVIEW

File: docs/CHANGES.rst [hunk 1]
```diff
@@ -10,13 +10,17 @@ Breaking changes:
 
 New features:
 
-- *add item here*
+- Allow to only get local roles for groups by passing inherit=False to groups.get_roles()
```
Comment (adamcheasley) [inline docs/CHANGES.rst]: I would rephrase this as follows:
"Passing inherit=False to groups.get_roles() will only get local roles for the group."

File: src/plone/api/group.py [hunk 2]
```diff
@@ -219,12 +219,23 @@ def get_roles(groupname=None, group=None, obj=None):
...
+    if obj is None:
+        return group.getRoles()
+    elif inherit:
+        # when context obj is available we bypass getRolesInContext method
+        # from PloneGroup class to use PloneUser class implementation because
+        # PloneGroup class disables all local roles support
+        # see: Products.PlonePAS.plugins.group.PloneGroup
+        roles = super(group.__class__, group).getRolesInContext(obj)
+        return list(roles)
+    elif obj and not inherit:
```
Comment (adamcheasley) [inline src/plone/api/group.py]: Could this not just be `else`?

Comment (pbauer) [inline src/plone/api/group.py]: Done

File: docs/CHANGES.rst (hunk 1 above)
Comment (pbauer) [inline docs/CHANGES.rst]: Done
